vocabularies/PersonalData.xml

```diff
@@ -142,9 +142,8 @@ Terms for contact and address information are defined in the [Communication voca
               <PropertyValue Property="Value" String="PurposeID" />
               <Annotation Term="Core.Description" String="The unique identifier for the purpose of a consent" />
```

Comment:
```suggestion
              <Annotation Term="Core.Description" String="The unique identifier for the purpose of a processing of personal data" />
```
The Description speaks of the "purpose of a consent", but the LongDescription speaks of the "purpose of data processing". This is confusing.